README.md

```diff
@@ -11,6 +11,11 @@ Ansible based project to configure a multi-node tsuru cluster.
 > Note: can be any number of nodes, but every section in the inventory file has to have at least one node. 
 > (it is possible to use a node in more that one section). 
 
```

Comment:
Would be nice if we add the librarian dependency with required version to be used here.